google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/v1/stub/SpannerStubSettings.java

```diff
@@ -299,9 +299,7 @@ public String extractNextToken(ListSessionsResponse payload) {
 
             @Override
             public Iterable<Session> extractResources(ListSessionsResponse payload) {
-              return payload.getSessionsList() != null
-                  ? payload.getSessionsList()
-                  : ImmutableList.<Session>of();
+              return payload.getSessionsList() != null;
```

Comment:
Typo here.

Reply:
fixed, waiting for tests to run and verify no other issues.